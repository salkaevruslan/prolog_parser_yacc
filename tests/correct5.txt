f :- first, second; third, extra (hello ((world))) wot, he.
a b c.
a (b c).