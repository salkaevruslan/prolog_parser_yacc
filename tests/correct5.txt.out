relation
	head
		identifier = f
	body
		disjunction
			conjunction
				identifier = first
				identifier = second
			conjunction
				identifier = third
				conjunction
					atom
						identifier = extra
						atom
							identifier = hello
							atom
								atom
									identifier = worldidentifier = wot
					identifier = he
relation
	head
		atom
			identifier = a
			identifier = b
			identifier = c
relation
	head
		atom
			identifier = a
			atom
				identifier = b
				identifier = c
